### PR TITLE
Feat : Added Grids

### PR DESCRIPTION
**Pull request by srejitk**
This Feature Request Adds Grid Layouts

1. 2 Column Grid
2. 3 Column Grid

image
image

### CODE REVIEW

**Review by nakulsharma15 — APPROVED** [review]
Looks good to me. No changes suggested!

File: stylesheets/section.css [hunk 1]
```diff
@@ -34,4 +34,14 @@
...
         font-family: "Roboto";
         outline: 2px solid;
+      }
+
+
+      .selected{
+        border-radius: 8px;
+        border: 2px solid black;
+      }
+      .selected:hover{
```
Comment (vaibhav-mahalle) [inline stylesheets/section.css:44]: try using variables for colors.

**Review by coldpigli — APPROVED** [review]
Everything looks great. Very intuitive class names.

**Review by vaibhav-mahalle — APPROVED** [review]
added some suggestions, everything else looks good.

File: stylesheets/section.css (hunk 1 above)
Comment (srejitk) [reply stylesheets/section.css:44]: Corrected in the latest commit. Thank you for the suggestion.

**Pull request merged**